aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -166,6 +169,21 @@ module aptos_framework::aptos_governance {
         voting_duration_secs: u64,
     }
 
+    struct GovernancePermission has copy, drop, store {}
+
+    /// Permissions
+    inline fun check_signer_permission(s: &signer) {
```

Comment:
maybe better to name these by the permission - i.e. check_governance_permission vs check_signer_permission